frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -76,11 +100,19 @@ const EditRecipeForm = ({ history }) => {
           </select>
         </div>
       </div>
-      <div id="recipe-submit-containaner">
-        <button type="submit" id="recipe-submit-btn" value="submit" onClick={handleSubmit}>Submit</button>
-      </div>
-      <div className="row form-footer">
-        <Footer />
+      <div>
+        <Ingredients
+          ingredients={ingredients}
+          handleIngredientChange={handleIngredientChange}
+          handleIngredientAdd={handleIngredientAdd}
+          handleIngredientRemove={handleIngredientRemove}
```

Comment:
What do you think about passing in `setIngredients` and doing these `handleChange`, etc. functions in the ingredients component?